### PR TITLE
Replaced maps with FlatList in AccountPickScreen.

### PR DESCRIPTION
**Pull request by vishwesh3**
Also fix: key error.
screen shot 2017-07-13 at 10 20 57 am

### CODE REVIEW

File: src/account/AccountList.js [hunk 1]
```diff
@@ -18,16 +18,20 @@ export default class AccountList extends React.PureComponent {
...
-            showDoneIcon={i === 0 && auth.apiKey !== '' && auth.apiKey === account.apiKey}
-            {...account}
-            onSelect={onAccountSelect}
-            onRemove={onAccountRemove}
-          />,
-        )}
+        <FlatList
+          keyboardShouldPersistTaps="always"
+          data={accounts}
+          keyExtractor={item => `${item.email}${item.realm}`}
```
Comment (vishwesh3) [inline src/account/AccountList.js]: Can't use apiKey as there may be cases where two accounts have empty apiKey (logged out ones). Can't use email as one email can be registered on different realms. Can't use realm as there may be multiple accounts with same realm. So combining email and realm.

Comment (borisyankov) [inline src/account/AccountList.js]: Good idea.

File: src/account/AccountList.js [hunk 2]
```diff
@@ -18,16 +18,20 @@ export default class AccountList extends React.PureComponent {
...
+          renderItem={({ item, index }) =>
+            <AccountItem
+              key={`${item.email}${item.realm}`}
```
Comment (borisyankov) [inline src/account/AccountList.js]: We don't need a `key` property on the child item.
I saw we use it in few other places where we do have FlatList you can remove it from these places too.

Comment (vishwesh3) [inline src/account/AccountList.js]: Done :+1:

**Pull request merged**